### DISCUSSION TITLE
nwg-look 0.2.4 (2023-07-26)

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#157] [Changelog]
https://github.com/nwg-piotr/nwg-look/releases/tag/v0.2.4

**Comment by j1nxie** [comment]
i'm not sure if this is the right place to have a feature request, but anyway,

is it possible to have a command-line option to export nwg-look preferences to files? i use nwg-look to set my themes, and on top of that, i have darkman for automatic dark mode toggling. i set my theme to a light one during the day, and a dark one when it's night time, and apps under XWayland reads the gtk settings files instead of directly reading from gsettings, so having a command-line option for nwg-look to automatically export to files would be really neat!

**Reply by nwg-piotr** [reply]
It should be doable. I'll take a closer look at it soon.

**Comment by nwg-piotr** [comment]
Copied to the issue tracker.
